web3/utils/module_testing/fallback_contract.py

```diff
@@ -0,0 +1,25 @@
+# Contract source code
+# 		contract A {
+#           uint data;
+#           function() { data = 1; }
+#           function getData() returns (uint r) { return data; }
+#       }
+FALLBACK_BYTECODE = (
+    "6060604052341561000f57600080fd5b60ae8061001d6000396000f300606060405260043610603f"
+    "576000357c0100000000000000000000000000000000000000000000000000000000900463ffffff"
+    "ff1680633bc5de30146053575b3415604957600080fd5b6001600081905550005b3415605d576000"
+    "80fd5b60636079565b6040518082815260200191505060405180910390f35b600080549050905600"
+    "a165627a7a723058204233f675dc222d76091504fcf32c45f024c9154ad12ae3bc0b411ab33ddc59"
+    "840029"
+)
+
+FALLBACK_ABI = [
```

Comment:
I don't think this module is needed.  There isn't anything special about testing fallback functions that would require us to test them across the various integration backends.  Can you remove this file?

Reply:
Sure, actually later I found out this is not needed but forgot to remove it...